polkadot/xcm/pallet-xcm/src/benchmarking.rs

```diff
@@ -385,6 +385,38 @@ benchmarks! {
 		let versioned_assets = VersionedAssets::V4(asset.into());
 	}: _<RuntimeOrigin<T>>(claim_origin.into(), Box::new(versioned_assets), Box::new(VersionedLocation::V4(claim_location)))
 
+	add_authorized_alias {
+		let origin = RawOrigin::Root;
+		let origin_location: VersionedLocation =
+			T::ExecuteXcmOrigin::ensure_origin(origin.clone().into()).unwrap().into();
+		let mut existing_aliases = BoundedVec::<VersionedLocation, MaxAuthorizedAliases>::new();
+		// prepopulate list with `max-1` aliases to benchmark worst case
+		for i in 1..MaxAuthorizedAliases::get() {
+			let alias = Location::new(1, [Parachain(i), AccountId32 { network: None, id: [42_u8; 32] }]).into();
+			existing_aliases.try_push(alias).unwrap()
+		}
+		AuthorizedAliasesMap::<T>::insert(&origin_location, existing_aliases);
+		// now benchmark adding new alias
+		let aliaser: VersionedLocation =
+			Location::new(1, [Parachain(1234), AccountId32 { network: None, id: [42_u8; 32] }]).into();
+	}: _(origin, Box::new(aliaser))
+
+	remove_authorized_alias {
+		let origin = RawOrigin::Root;
+		let origin_location: VersionedLocation =
+			T::ExecuteXcmOrigin::ensure_origin(origin.clone().into()).unwrap().into();
```

Comment:
instead of `ensure_origin`, I would use `try_origin` or `try_successful_origin` to be aligned with other pallet_xcm benchmarks where we use `T::ExecuteXcmOrigin::`, so if a runtime does not allow `ExecuteXcmOrigin`, avoid benchmarking failure - unless we want to require them to enable `ExecuteXcmOrigin` :)

E.g.:
```
let origin_location = T::ExecuteXcmOrigin::try_origin(send_origin.clone().into())
			.map_err(|_| BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)))?;
```
or
```
let execute_origin =
			T::ExecuteXcmOrigin::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
```